changelogs/fragments/2124-consul_kv-pass-token.yml

```
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - plugins/lookup/consul_kv.py - use self.get_option and
+    wire the exsisting token attribute into consul_api call.
+    (https://github.com/ansible-collections/community.general/issues/2124)
```

hi @fkuep, this changelog fragment will later be assembled into something like this: https://github.com/ansible-collections/community.general/blob/stable-2/CHANGELOG.rst#v2-3-0


```suggestion
  - consul_kv lookup - use ``self.get_option`` and
    wire the existing ``token`` attribute into consul_api call
    (https://github.com/ansible-collections/community.general/issues/2124).
```

---

Thanks, I'll take the change!  
And thanks for showing; I did not not know that You can escpape markdown into these message.  
Without prior experiance , I am guessing myself through the meta-data-with this little code  change.